src/deadline_worker_agent/sessions/session.py

```diff
@@ -1084,33 +1102,63 @@ def _action_updated_impl(
             ActionState.TIMEOUT,
         )
 
+        if self._output_sync_target_action is not None:
+
+            if OPENJD_ACTION_STATE_TO_DEADLINE_COMPLETED_STATUS.get(action_status.state, None):
+                task_run_action = self._output_sync_target_action
+                self._output_sync_target_action = None
+                logger.info(
+                    f"task run is {task_run_action}, _output_sync_target_action is {self._output_sync_target_action}"
+                )
+                self._handle_action_update(is_unsuccessful, action_status, task_run_action, now)
+            else:
+                logger.info("AttachmentUploadAction is still running")
```

Comment:
Should this be logged at `INFO` level? In the future when output upload support progress reporting, this will get logged every time there is a progress update.

Reply:
I'll change to `DEBUG`.